### PR TITLE
Ronald branch

### PR DESCRIPTION
**Pull request by ronaldhoang**
uhhh I think this is the updated one with the working send message page without the working send message

### CODE REVIEW

**Comment by ronaldhoang** [comment]
uhhhhhhh ignore the multiple commits that was just me figuring out github

File: app/src/main/java/example/jackliu/nonewfriends/sendMsg.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.message_page);
+        sendbutton = (Button) findViewById(R.id.sendButton) ;
+
+
+
+
+        sendbutton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v){
```
Comment (jackjunliu) [inline app/src/main/java/example/jackliu/nonewfriends/sendMsg.java:25]: wait so all this does is just change the page when the button is pressed?

File: app/src/main/java/example/jackliu/nonewfriends/sendMsg.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+                startActivity(new Intent(sendMsg.this, sentMsgScreen.class));
+                finish();
```
Comment (jackjunliu) [inline app/src/main/java/example/jackliu/nonewfriends/sendMsg.java]: also, remove this finish(), or else the back button won't work

File: app/src/main/java/example/jackliu/nonewfriends/HomeActivity.java [hunk 3]
```diff
@@ -87,6 +106,7 @@ public void onAuthStateChanged(@NonNull FirebaseAuth firebaseAuth) {
                     // user auth state is changed - user is null
                     // launch login activity
                     startActivity(new Intent(HomeActivity.this, LoginActivity.class));
+                    finish();
```
Comment (jackjunliu) [inline app/src/main/java/example/jackliu/nonewfriends/HomeActivity.java:106]: > finish();

Remove this one too

File: app/src/main/java/example/jackliu/nonewfriends/HomeActivity.java [hunk 4]
```diff
@@ -42,12 +43,30 @@ protected void onCreate(Bundle savedInstanceState) {
...
                 startActivity(new Intent(HomeActivity.this, MapsActivity.class));
+                finish();
+            }
+        });
+
+        sendMsg.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v){
+                startActivity(new Intent(HomeActivity.this, sendMsg.class));
+                finish();
```
Comment (jackjunliu) [inline app/src/main/java/example/jackliu/nonewfriends/HomeActivity.java]: Remove all the finish();

**Review by jackjunliu — CHANGES_REQUESTED** [review]

File: app/src/main/java/example/jackliu/nonewfriends/sendMsg.java (hunk 1 above)
Comment (ronaldhoang) [reply app/src/main/java/example/jackliu/nonewfriends/sendMsg.java:25]: at the moment yes, as we're figuring out how the message feature actually works

**Pull request closed without merging**